app/controllers/api/v1/auth_tokens_controller.rb

```diff
@@ -6,7 +6,7 @@ class AuthTokensController < API::V1::BaseController
       before_action :set_user
 
       def create
-        Forms::AuthTokens::Create.new(user: @user).save
+        Forms::AuthTokens::Create.new(resource: @user).save
```

Comment:
We should have a separate route for the option where you do `POST /api/v1/organizations/:organization_id/auth_tokens` right?